### PR TITLE
Adding profile region option

### PR DESCRIPTION
**Pull request by kubihie**
I added `--region` and `--profile` command option.

### CODE REVIEW

File: aq/engines.py [hunk 1]
```diff
@@ -23,17 +23,22 @@ def __init__(self, options=None):
...
         self.last_refresh_time = defaultdict(int)
 
-        self.boto3_session = boto3.Session()
+        self.boto3_session = boto3.Session(profile_name=self.profile)
         # dash (-) is not allowed in database name so we use underscore (_) instead in region name
         # throughout this module region name will *always* use underscore
-        if self.boto3_session.region_name:
+        if self.region:
+            self.default_region = self.region.replace('-', '_')
+        elif self.boto3_session.region_name and self.region is None:
```
Comment (lebinh) [inline aq/engines.py]: @kubihie the check for `self.region is None` seems redundant to me here, isn't it?

**Comment by lebinh** [comment]
thanks @kubihie 👍 

The initial idea is to let these settings be controlled by user in their boto or AWS CLI settings but this seems good to have too.

**Comment by kubihie** [comment]
It's exactly as you say.
It has been deleted redundant:smile:

**Comment by ebpitts** [comment]
I have two named profiles configured in my `credentials` file. To avoid mistakes I do not have a default profile. To use them with the `aws` cli, I use the `--profile` option to choose one. I am unable to use `aq` because it does not let me specify the profile.

**Comment by lebinh** [comment]
@ebpitts I am merging this so you can try to use `aq` from master branch for your use case
@kubihie sorry for the delay and thank you for this contribution 👍

**Comment by ebpitts** [comment]
Thanks! I installed from master and it's working.

**Pull request merged**